ui/cypress/integration/configuration.test.ts

```diff
@@ -0,0 +1,59 @@
+describe('Configuration', () => {
+  let srcCredentials: any
```

Comment:
a nit: Credentials are conventionally used for authentication data such as username+password, certificate, session. This variable  holds a bit more,  its semantics could be better `sourceConfig` or `sourceFixture` ... "naming is only the two hard things"